### ISSUE TITLE
Suggestion: Retouch colors and maybe add some more

### ISSUE THREAD

**Issue by artemisYo** [casperstorm/ferra#4]
# TL;DR
Colors (sage in particular) seem off to me; I propose slight adjustments, maybe adding more colors too.
The following section *Explanation* is a bunch of writing, explaining my thoughts; **I recommend skipping it if you're not interested.**
# Explanation
## Motivation
Ferra has a rich earthy tone, partly because of the abundance of browns and oranges used in the scheme. However, this comes at the cost of seeming overly homogeneous. One way to combat it is to put slightly more attention towards the accentual colors present. 
Another noticeable outcome of the lacking focus on accentual colors is that they do not fit in as well as they could.
## Notes
One thing that is easy to notice is the general lack of consistent saturation or chroma in the accentual colors; while Mist is practically white, Ember and Honey stun with high chromaticity. It would also be beneficial to include more variants of said color, to be able to better cover the need for brighter and darker variations, this goes hand in hand with another easily apparent problem: the colors do not have a consistent luminosity either.
## Solution
In the following, I have established 8 categories of accentual colors, most with 3 grades of luminosity. I tried matching the chroma within a category, so as to ensure cohesion.
The chroma between color categories has outliers, but also tries to stay within bounds.[^1]
The luminosity of the grades tries to stay consistent over categories. Furthermore, I propose shifting the base colors (Ash, Umber and Bark) to be slightly more chromatic, with a bigger lean to brown.
Additionally, I added a blue and purplish color to fit the common Base16 color schemes more; while I personally dislike said schemes, it could be beneficial to allow for more color variety.
I am still undecided on whether green should retain the chroma levels present in Sage, or to increase them.[^2]
[^1]: the orange, red and yellow categories having nearly the same chroma, with green and white being low-chroma and rose being somewhere in-between.
[^2]: approximately double the previous chroma
[^3]: Here the vivid and light orange are Coral and Blush respectively, with Blush chroma-adjusted to fit Coral. The vivid low chroma green is Sage without any changes. Vivid rose is Rose, dark red is Ember, light yellow is honey.

# The Actual Adjustments
A picture of the adjusted colors in helix using the old Sage color; the change is minimal but perceptible:
![helix](https://github.com/casperstorm/ferra/assets/106774277/9c03bbfb-33fa-4920-beb5-959897412f29)
The colors[^3] enumerated:
<table>
  <tr>
    <th> Color </th> 
    <th> Dark </th> 
    <th> Vivid </th>
    <th> Light </th>
  </tr>
  <tr>
    <td> Orange </td>
    <td> 
<img src="https://github.com/casperstorm/ferra/assets/106774277/ff212246-917c-4885-8fe7-bbc70487819b" width="25rem" /> 
      #d67751
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/1f612f18-0106-4273-9c43-a8220da3b038" width="25rem" />
      #ffa07a
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/7548c0a0-e4c4-4117-b6fe-2f7abb5bbc19" width="25rem" />
      #ffc49e
    </td>
  </tr>
  <tr>
    <td> Green (low chroma) </td>
    <td>   
<img src="https://github.com/casperstorm/ferra/assets/106774277/a4737310-a62a-4223-9fb5-b4962ea464f0" width="25rem" />
      #8b906f
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/15152322-f5c3-49ae-a7ac-6f1c59de4116" width="25rem" />
      #b1b695
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/decb38f3-38c1-46f0-bbe2-e74697ad25c7" width="25rem" />
      #d6dbba
    </td>
  </tr>
  <tr>
    <td> Green (high chroma) </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/a5f45308-1917-421b-b877-908c81f73ce6" width="25rem" />
      #8c9368
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/97fb5412-ea88-40c7-ace8-e714e35f634a" width="25rem" />
      #b2b98e
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/2566463f-608b-4163-a7b1-e46eab7ed4d7" width="25rem" />
      #d7deb3
    </td>
  </tr>
  <tr>
    <td> Red </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/adc6cfd9-31d1-4ef2-bf98-6d0a42a9f2fd" width="25rem" />
      #e06b75
    </td>
    <td>
      None
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/bc083f0a-88de-45a8-880c-a03f081abf61" width="25rem" />
      #ff919b
    </td>
  </tr>
  <tr>
    <td> Rose </td>
    <td>
      None
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/c09c2753-ffa4-4c3e-9dc5-e5eb714d2a72" width="25rem" />
      #f6b6c9
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/be59ef4f-2894-4762-be73-5e4656ae8ac7" width="25rem" />
      #ffc8db
    </td>
  </tr>
  <tr>
    <td> Yellow </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/efd65cf6-2dcc-4729-bec1-a288ad30a2a0" width="25rem" />
      #d8a442
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/2c8a50fa-7bd6-4009-a9d6-68e842d065cd" width="25rem" />
      #e9b553
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/bd7853b1-d637-42e2-9d70-0a1e19b66214" width="25rem" />
      #f5d76e
    </td>
  </tr>
  <tr>
    <td> White </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/940afbd3-777c-4500-8d68-cecd89b7b1df" width="25rem" />
      #90909d
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/d7f9d974-0bc7-4fe8-88dc-864a17171df5" width="25rem" />
      #b8b8c6
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/785dd058-6788-4ab0-95e0-192143cc4384" width="25rem" />
#d6d6e4
    </td>
  </tr>
  <tr>
    <td> Blue </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/07041276-5f01-4018-b4f3-09347b995a30" width="25rem" />
      #839ae3
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/6cdb1478-d1fa-4e72-bdec-2ce6f6191a26" width="25rem" />
      #9eb3ff
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/95ed5f1b-b855-4be4-a396-9cc1e88dae10" width="25rem" />
      #bbd0ff
    </td>
  </tr>
  <tr>
    <td> Purple </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/3f9e765c-09c4-4277-828f-38cc320648a7" width="25rem" />
      #ba87d8
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/b9cd01fe-c766-4c19-8616-c2e5075b1d9c" width="25rem" />
      #d69eff
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/87fa4464-0ff1-4c25-a76e-84871ad418ec" width="25rem" />
      #f6b3ff
    </td>
  </tr>
</table>

The adjustments to the base colors are as follows:
| Color | Hex |
| --- | --- |
| Night | Unchanged |
| Ash | <img src="https://github.com/casperstorm/ferra/assets/106774277/7bc3fa83-749c-4ae3-9dbd-0d6b6e63d748" width="25rem" /> #3c3538 |
| Umber | <img src="https://github.com/casperstorm/ferra/assets/106774277/cf79e170-c87e-4917-a67f-ef1c913e9454" width="25rem" /> #4c4245 |
| Bark | <img src="https://github.com/casperstorm/ferra/assets/106774277/1f654c01-4ca5-463d-90a4-c1d5f1f6cd0e" width="25rem" /> #6f5c5e |

**Comment by artemisYo** [comment]
Considering many other themes featuring light versions of their themes, it would also probably not hurt too much, to display colors for said light variant. Thus, iterating on this proposal, I have narrowed down the changes to only feature 2 shades of each color (per variant), merging rose and ember shades of one color. The following is the revised palette:

<table>
  <tr>
    <th> Base </th>
    <th> Dark Mode Colors </th>
    <th> Light Mode Colors </th>
  </tr>
  <tr>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/7558ac52-8693-481e-88ed-35742debf58b" height="70rem" />
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/7e028cd2-f49c-4881-a94b-e00b1203fb90" height="70rem" />
    </td>
    <td>
<img src="https://github.com/casperstorm/ferra/assets/106774277/9421ac93-01c0-41ff-b329-ae9e08af80c5" height="70rem" />
    </td>
  </tr>
</table>

<details>
<summary> Hex Codes </summary>
<table>
  <tr>
    <th></th>
    <th colspan=2> Dark Mode </th>
    <th colspan=2> Light Mode </th>
  </tr>
  <tr>
    <td> <b>Color</b> </td>
    <td> <b>Main Shade</b> </td>
    <td> <b>Alternate Shade</b> </td>
    <td> <b>Main Shade</b> </td>
    <td> <b>Alternate Shade</b> </td>
  </tr>
  <tr>
    <td> White </td>
    <td> #b8b8c6 </td>
    <td> #d6d6e4 </td>
    <td> #a4a4bb </td>
    <td> #7d7e93 </td>
  </tr>
  <tr>
    <td> Green </td>
    <td> #b2b98e </td>
    <td> #d7deb3 </td>
    <td> #9aa95d </td>
    <td> #758438 </td>
  </tr>
  <tr>
    <td> Yellow </td>
    <td> #e9b553 </td>
    <td> #f5d76e </td>
    <td> #d7a135 </td>
    <td> #a6781a </td>
  </tr>
  <tr>
    <td> Orange </td>
    <td> #ffa07a </td>
    <td> #ffc49e </td>
    <td> #ef8961 </td>
    <td> #c6623a </td>
  </tr>
  <tr>
    <td> Rose </td>
    <td> #ff919b </td>
    <td> #f6b6c9 </td>
    <td> #e99fb6 </td>
    <td> #d15262 </td>
  </tr>
  <tr>
    <td> Purple </td>
    <td> #d69eff </td>
    <td> #f6b3ff </td>
    <td> #c689f2 </td>
    <td> #c689f2 </td>
  </tr>
  <tr>
    <td> Blue </td>
    <td> #9eb3ff </td>
    <td> #bbd0ff </td>
    <td> #86a0f4 </td>
    <td> #6a88d8 </td>
  </tr>
</table>
With the 4 base colors being:
<table>
  <tr>
    <th> Color </th>
    <th> Dark Mode </th>
    <th> Light Mode </th>
  </tr>
  <tr>
    <td> Night (& Day?) </td>
    <td> #2b292d </td>
    <td> #a89594 </td>
  </tr>
  <tr>
    <td> Ash </td>
    <td> #a89594 </td>
    <td> #cbb6b4 </td>
  </tr>
  <tr>
    <td> Umber </td>
    <td> #4d424b </td>
    <td> #dec8c6 </td>
  </tr>
  <tr>
    <td> Bark </td>
    <td> #6f5d63 </td>
    <td> #edd8d6 </td>
  </tr>
</table>
</details>

**Comment by artemisYo** [comment]
Actually scratch that, my colors suck
